src/datamodel/game.rs

```diff
@@ -1,8 +1,9 @@
 use crate::datamodel::board::Board;
 use crate::datamodel::player::Player;
+use std::rc::Rc;
 
 pub struct Game {
     pub num_players: u32,
-    pub players: Vec<Player>,
+    pub players: Vec<Rc<Player>>,
```

Comment:
This change will impact https://github.com/tul1/avalon-rs/pull/19, which version are we keeping. Are you sure that we could use `mut&` instead?

Reply:
See my comment in that PR. I believe we should use `Rc` because it is a smart pointer which has ownership of the Player, but allows us to have several pointer to the same instance. We don't want to change it, so it should be immutable IMO. The other option would be to clone the player, which I think is not a good idea.